libs/hmpb/src/ballot_templates/nh_ballot_template.tsx

```diff
@@ -671,7 +671,6 @@ async function BallotPageContent(
     .filter((section) => section.length > 0);
 
   // Add as many contests on this page as will fit.
-  const contestSectionsLeftToLayout = contestSections;
```

Comment:
Unnecessary extra variable, and a bit confusing since we're mutating the underlying array anyway